releases/release-1.22/release-notes/maps/pr-100678-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 100678
+releasenote:
+  text: Add a new Priority and Fairness rule that exempts all probes (`/readyz`, `/healthz`,
+    `/livez`) to prevent restarting of `healthy` `kube-apiserver` instance by kubelet.
```

Comment:
Does `healthy` need to come in backticks ?

Reply:
I was pondering on this a lot. I chose to put in a code block because I thought it is meant for the status of the kube-apiserver. But now I see it is misleading, I will paraphrase the second part of the note.